app/api/packs/search/route.ts

```diff
@@ -35,8 +35,7 @@ export async function GET(request: NextRequest) {
 	const skip = (pageNumber - 1) * PAGE_SIZE
 
 	const where: Prisma.QuestionPackWhereInput = {
-		pending: false,
-		denied: false,
+		status: 'approved',
```

Comment:
This should be using the status enum. (don't forget to import it as it is not in this file)
```suggestion
		status: Status.approved,
```